core/bentley/src/BeEvent.ts

```diff
@@ -171,3 +171,24 @@ export class BeEventList<T extends Listener> {
     this._events[name] = undefined;
   }
 }
+
+/**
+ * Retrieves the listener type of a {@link BeEvent}.
+ * @note This can be used with a narrowed event type.
+ *
+ * @example
+ * ```ts
+ * const ev = new BeEvent<(x: number, y: string) => void>();
+ *
+ * // type EvListener = (x: number, y: string) => void
+ * type EvListener = ListenerType<typeof ev>;
```

Comment:
```suggestion
 * const event = new BeEvent<(x: number, y: string) => void>();
 * const callback: ListenerType<typeof event> = (x: number, y: string) => {
 *   console.log(`${x}, ${y}`);
 * };
```
Make example clearer.

Reply:
Applied suggestion without explicit arg types since that's the expected usage.